resource/template/actioncode/sample.js

```diff
@@ -1,3 +1,20 @@
+/*
```

Comment:
I'd liked to know the license is required for a sample code.
I am not sure about the licensing policy.

Reply:
This is according to the `scancode.sh`.
Even test codes in the openwhisk repo have a license header.
https://github.com/apache/openwhisk/blob/master/tests/dat/actions/echo.js

Reply:
This is not a test code. When a user creates a new action with the plugin, this sample code is used.

Reply:
I feel it's reasonable to add a license header as this code is also provided by asf?